site/content/docs/install.md

```diff
@@ -41,17 +42,7 @@ service docker start
 ```
 
 ### All Linux Distributions
-Follow the [Docker Install Instructions](http://docs.docker.com/installation/) for your distro/platform. 
-
-
-## Install SNAS Using Docker
-Each docker file contains a readme file, see below:
-
-* [All-In-One](install_aio)
-* [UI](install_ui)
-* [Collector](install_collector)
-* [Kafka](install_kafka)
-* [MySQL](install_mysql)
+Follow the [Docker Install Instructions](http://docs.docker.com/installation/) for your distro/platform.
```

Comment:
why remove the links.  @serpilb  has a fix to ensure they open correctly.   What needs to be added is an update to the sidebar selection based on the relative link selection.